legacy/src/api/payment-requests/payment-requests.md

```diff
@@ -283,6 +283,7 @@ Payment Activities are created when a Payment Request has been **created**, **pa
 | PAYMENT_DECLINED             | The payment parameters were valid but payment was declined because additional payment restrictions were violated. For example, asset not active, asset overdrawn, quota exceeded or line item category restrictions. |
 | PAYMENT_REQUEST_EXPIRED      | The payment request has expired.                                                                                                                                                                                     |
 | NO_AVAILABLE_PAYMENT_OPTIONS | No payment options match the requested payment parameters.                                                                                                                                                           |
+| INACTIVE_ASSET               | The asset on the payment request is inactive.                                                                                                                                                                        |
```

Comment:
@seliwe I don't think we should document assets as "being on the Payment Request". 

Elsewhere we've got `The asset is not spendable. It may have been disabled, expired, or already spent.`

Reply:
I'll update to 'The asset used to pay the payment request is inactive.'